test/html/HtmlConverterTest.php

```diff
@@ -1,14 +1,77 @@
 <?php
 require_once 'src/wovnio/html/HtmlConverter.php';
 require_once 'src/wovnio/html/HtmlReplaceMarker.php';
-
+require_once 'src/wovnio/wovnphp/Utils.php';
+require_once 'src/wovnio/wovnphp/Headers.php';
 require_once 'src/wovnio/modified_vendor/simple_html_dom.php';
 
 use Wovnio\Html\HtmlConverter;
+use Wovnio\Wovnphp\Url;
+use Wovnio\Wovnphp\Utils;
 use Wovnio\Html\HtmlReplaceMarker;
 use Wovnio\ModifiedVendor\simple_html_dom;
 
 class HtmlConverterTest extends PHPUnit_Framework_TestCase {
+  private function getEnv($num="") {
+    $env = array();
+    $file = parse_ini_file(dirname(__FILE__) . '/../mock_env' . $num . '.ini');
+    $env = $file['env'];
+    return $env;
+  }
+
+  /**
+    * @group new
+   */
+  public function testHreflang() {
```

Comment:
We need to add tests to check that chinese hreflang values are correctly set with ISO 639 spec.

Reply:
Needs tests with custom lang aliases too.